project/SparkBuild.scala

```diff
@@ -420,6 +422,16 @@ object Unidoc {
   )
 }
 
+object Scalastyle {
```

Comment:
Thanks for looking! if anyone has an easy fix, great, but worst case I will just not move the file. It's only one of the secondary goals here.